app/controllers/api/v1/cars.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CarsController < ApplicationController
+  before_action :set_cars, only: %i[show destroy]
+  before_action :set_user, only: %i[create destroy]
+
+  # GET /cars/1
+  def show
+    render json: @car
+  end
+
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_car
+    @car = car.find(params[:id])
+  end
+
+  def set_user
+    @user = current_user
+  end
```

Comment:
Since this project isn't using devise, this function would not work. The user id would have to be gotten from the URL. Please remove this function